sass/patternfly.scss

```diff
@@ -59,6 +59,24 @@ $fa-square-o: "\f096";
   background: $color-pf-black-100;
 }
 
+.btn-transparent {
```

Comment:
@nicpuddu is this the proper file to put this sytles in? Just spotted there is a patternfly-override.scss file as well

Reply:
yeah it is fine